src/tools/analyzer/source_files.py

```diff
@@ -0,0 +1,58 @@
+from enum import Enum
+import configparser
+from os import listdir
+from os.path import isfile, join
+import glob
+
+
+class Reader:
+    class Component(Enum):
```

Comment:
If `class Component(Enum)` should not be some common thing included here and in `source_journald.py` rather be defined twice?

Reply:
It could be defined once underneath `class Analyzer` in module_request.py instead, is that what you had in mind?